mbuild/packing.py

```diff
@@ -144,6 +147,11 @@ def fill_box(
         Number of compounds to be filled in box.
     box : mb.Box
         Box to be filled by compounds.
+    use_pbc : bool, default=False
```

Comment:
I think we can remove this as an argument. I think it should be automatically be handled from setting self.periodicity. May have to require either all or None periodicity, depending on if PackMol can handle that.

Reply:
Okay replying to my own comment here. I think we actually need an argument here, since typically you're passing the box as an object here, and single compounds, so periodicity isn't always set. However, the result of this argument should definitely set compound.periodicity so we should check that.

I also think it would be nice if you could pass a list of Bools of size 3, [True, True, False]. Sort of thing. Again, not sure how that looks on the packmol side.

Reply:
After reading through the documentation I don't think a list of bools would work, it's all or nothing unless I'm mistaken.

Reply:
Thanks for checking!